tools/MINERVA/upload_data_overlay.xml

```diff
@@ -0,0 +1,24 @@
+<tool id="minerva" name="MINERVA Upload Data Overlay" version="14.0">
+    <description>MINERVA API uploading overlays </description>
+    <command detect_errors="exit_code"><![CDATA[
+python '$__tool_directory__/upload_data_overlay.py'
+--login '$login'
+--password '$password'
+--overlay '$overlay'
+--api_url '$api_url'
+--project_id '$project_id' > '$log'
+    ]]></command>
+    <inputs>
+        <param name="login" type="text" label="Login name" />
+	<param name="password" type="text" label="Password" />
+	<param name="overlay" type="data" format="tabular" label="Overlay file"/>
+	<param name="api_url" type="text" label="API url" />
+	<param name="project_id" type="text" label="Project ID" />
+    </inputs>
+    <outputs>
+        <data name="log" format="txt" label="${tool.name} on ${on_string}: Log" />
+    </outputs>
+    <help><![CDATA[
```

Comment:
I guess tests for this tool are hard, but maybe we can add a test on the log file to ensure that everything ran ok on the Galaxy end? (e.g. certain expected log output is present)